runtime/src/bank.rs

```diff
@@ -6108,6 +6108,15 @@ impl Bank {
         Some(vote_account.clone())
     }
 
+    /// Get the EpochStakes for the current Bank::epoch
+    pub fn current_epoch_stakes(&self) -> &EpochStakes {
```

Comment:
can this function be private?

Reply:
yeah it can, good catch. would you prefer I fix this and `iproduct!` in a master commit and then backport both commits together? or merge this and then fix in master and backport separately?

Reply:
I'm okay with either approach. Neither is a big enough deal that I'd block the PR